Utils/helper_functions.py

```diff
@@ -0,0 +1,60 @@
+# functions to find files, get median,
+
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+# function to open a file and return the data in a pandas dataframe
+def open_file(file_path):
+    df = pd.read_csv(file_path, sep=",")
+    return df
+
+
+# function to find the median row of a column in a dataframe and return the value and index of the row
+def find_median(df, column):
+    median = df[column].median()
+    median_index = df[column].sub(median).abs().idxmin()
+    print(median_index)
+    return median, median_index
+
+
+# function to plot a histogram of a column in a dataframe and save the plot as a png file given a row name and column name
+def plot_histogram(
+    df: pd.DataFrame,
+    feature_name,
+    image_name_column="image_name",
+    row_name=None,
+    histogram_=False,
+    bins=50,
```

Comment:
Consider keeping the type hints for this function consistent by either providing them for all or none of the parameters.